bee-api/bee-rest-api/src/types.rs

```diff
@@ -170,8 +166,7 @@ impl TryFrom<&Message> for MessageDto {
     fn try_from(value: &Message) -> Result<Self, Self::Error> {
         Ok(MessageDto {
             network_id: value.network_id().to_string(),
-            parent_1_message_id: value.parent1().to_string(),
-            parent_2_message_id: value.parent2().to_string(),
+            parents: vec![value.parents()[0].to_string(), value.parents()[1].to_string()],
```

Comment:
that's probably a bug. From what @Thoralf-M  pointed out in the discussion on Discord (parents now need to be unique, there are cases where a message only has 1 parent like messages referencing the genesis) the index 1 might not exist, hence this code will panic in such a special situation.

Reply:
okey if so, then we need to adjust also `two_non_lazy_tips()` from the urts.rs. Should I take care of this? @thibault-martinez

Reply:
Yes, that was from the point where we thought it will be fixed to 2 parents for the first release

Reply:
@rufsam Please do

Reply:
The REST API of Hornet still returns 2 tips https://github.com/gohornet/hornet/blob/develop/plugins/restapi/v1/types.go#L36
So is this not implemented yet?
Also, how many `MessageId`s should `tangle::get_messages_to_approve()` return? 1? 8?

Reply:
@rufsam look at the other branch https://github.com/gohornet/hornet/blob/feat/multiple-parents/plugins/restapi/v1/types.go

Reply:
okey thanks, seems like Hornet currently returns 4 tips from the tip pool <https://github.com/gohornet/hornet/blob/04a47e97326bd0b74e4b078a720ad54ebf628bea/pkg/tipselect/urts.go#L329>